The YMZ ADPCM emulation added upstream is in the CMake build but not the libretro
makefile, so the core fails to load with undefined ymzadpcm_* symbols.

--- a/sdl/Makefile.common
+++ b/sdl/Makefile.common
@@ -77,7 +77,9 @@
 	$(wildcard $(NP2_PATH)/mem/*.c) \
 	$(wildcard $(NP2_PATH)/sdl/*.c) \
 	$(NP2_PATH)/sound/adpcmc.c \
-	$(NP2_PATH)/sound/adpcmg.c \
+	$(NP2_PATH)/sound/adpcmg.c \
+	$(NP2_PATH)/sound/ymzadpcmc.c \
+	$(NP2_PATH)/sound/ymzadpcmg.c \
 	$(NP2_PATH)/sound/beepc.c \
 	$(NP2_PATH)/sound/beepg.c \
 	$(NP2_PATH)/sound/cs4231c.c \
